styles/_contribute.scss

```
@@ -1041,14 +1041,17 @@
                     /* 150% */
                     margin-top: 15px;
                     margin-bottom: 15px;
+                    a{
+                        color: #9A52FF;
```

This is a color we have defined in `_colors.scss` so let's use the variable instead of having another random constant:

```suggestion
                        color: $blue-purple-400;
```